FIXME code assuemes that mbstowcs() != -1 is a reliable way to check for
invalid characters but on musl libc this only works for when locale is
UTF-8

--- a/src/main.c
+++ b/src/main.c
@@ -547,6 +547,8 @@
 
 int main(int argc, char **argv)
 {
+	cdw_locale_init();
+
 	if (argc > 1) {
 		/* if ((strcmp((char *) argv[1], "--help") == 0) || (strcmp((char *) argv[1], "-h") == 0)) { */
 		/* 2TRANS: this is message printed in terminal,
